toolkit/javascripts/clear-filters.js

```diff
@@ -23,6 +23,7 @@
     this.$clearFilterLink.on('click', function(event) {
         event.preventDefault();
         _this.clearFilters();
+        $(this).blur();
```

Comment:
Bit worried about this - what happens if I am a keyboard user, and I want to tick three filters in a row? Does this change mean I have to tab through all the filters again to get back to where I was?